mmv1/products/networksecurity/FirewallEndpointAssociation.yaml

```diff
@@ -14,7 +14,7 @@
 --- !ruby/object:Api::Resource
 name: 'FirewallEndpointAssociation'
 base_url: '{{parent}}/locations/{{location}}/firewallEndpointAssociations'
-create_url: '{{parent}}/locations/{{location}}/firewallEndpointAssociations?firewallEndpointId={{name}}'
+create_url: '{{parent}}/locations/{{location}}/firewallEndpointAssociations?firewallEndpointAssociationId={{name}}'
```

Comment:
That was just never right, huh.

Reply:
Yes, this is how it should have been, but maybe the tests got skipped at one point.